lib/rexml/source.rb

```diff
@@ -76,8 +76,12 @@ def match(pattern, cons=false)
       end
     end
 
-    def string=(string)
-      @scanner.string = string
+    def pos
+      @scanner.pos
+    end
+
+    def pos=(pos)
```

Comment:
How about avoid abbreviation here? The original word isn't so long.

```suggestion
    def position
      @scanner.pos
    end

    def position=(position)
```